torch/hub.py

```diff
@@ -446,6 +446,7 @@ def load_state_dict_from_url(url, model_dir=None, map_location=None, progress=Tr
     Args:
         url (string): URL of the object to download
         model_dir (string, optional): directory in which to save the object
+        file_name (string, optional): To name the downloaded file and dodge any confusion
```

Comment:
I'd mention "If `file_name` is not set, the original filename will be kept."

Reply:
i think (optional) send the same message

Reply:
i think (optional) send the same message

Reply:
Hmmm I guess that's fine. 
Please rebase on top of master to make sure CI passes, thanks!